app/components/post_list/post_list.js

```diff
@@ -129,7 +130,7 @@ export default class PostList extends PureComponent {
             this.shouldScrollToBottom = false;
         }
 
-        if (!this.hasDoneInitialScroll && this.props.initialIndex > 0 && this.state.contentHeight) {
+        if (!this.hasDoneInitialScroll && this.props.initialIndex > 0 && this.state.contentHeight > 53) {
```

Comment:
What does `53` represent? A limit to post content height?

Reply:
I can make  this  a constant,  but 53  is the height of the  loading indicator when the post list is empty